### DISCUSSION TITLE
Run script with having to answer Y about restarting MySQL

### DISCUSSION THREAD

**Discussion by develand-acams** [Releem/releem-agent#227] [Feature Requests]
Hi, I was thinking about a cron job that runs every couple days that would apply Releem changes automatically. That way it stays tuned for optional performance without me remembering to run the script. It would be running in the middle of the night so as to not disturb anyone, although the restart process only takes about a second anyway.

Is there an option to run the script and automatically respond Y when prompted, or maybe an argument to the script that would bypass that? Like -Y added to the end.

**Comment by drupaladmin** [comment]
Hi @develand-acams, thanks for suggestion. 

you could use this command:
RELEEM_RESTART_SERVICE=1 /opt/releem/mysqlconfigurer.sh -a

The same feature request is here: https://github.com/Releem/mysqlconfigurer/discussions/187#discussioncomment-7482136

Added issue to add this command in documentation: https://github.com/Releem/mysqlconfigurer/issues/228
